### PR TITLE
Samr21: autostart gdb when running make debug

### PR DESCRIPTION
**Pull request by x3ro**
:penguin:

### CODE REVIEW

**Comment by thomaseichinger** [comment]
```bash
RIOT/boards/samr21-xpro/dist/gdb.conf: No such file or directory.
```
Please add the referenced file or remove the argument.

**Comment by x3ro** [comment]
Added.

File: boards/samr21-xpro/dist/gdb.conf [hunk 1]
```diff
@@ -0,0 +1,3 @@
+target remote localhost:3333
+monitor reset halt
+load
```
Comment (thomaseichinger) [inline boards/samr21-xpro/dist/gdb.conf]: As flashing is quite slow on this board we should maybe omit the load command?

Comment (x3ro) [inline boards/samr21-xpro/dist/gdb.conf]: To be honest, I'm not entirely sure what load does exactly. I was under the impression that it loads debug symbols from the device or something akin. Is that correct?

Comment (gebart) [inline boards/samr21-xpro/dist/gdb.conf]: No debug symbols are stored on the device. `load` loads the program binary onto the target (i.e. flashing the device).
`file` loads debug symbols etc into GDB from the file system, but it is not necessary to give that command if you already gave the filename as a command line argument when starting GDB.

Comment (x3ro) [inline boards/samr21-xpro/dist/gdb.conf]: @gebart Thanks for the explanation :smile:

**Comment by x3ro** [comment]
`load` removed.

**Comment by thomaseichinger** [comment]
ACK & Go

**Pull request merged**